.gitignore

```diff
@@ -60,6 +60,9 @@ yarn.lock
 # dotenv environment variables file
 .env
 
+# json secrets
+*.secrets.json
```

Comment:
I have the secrets stored in a json file, but I can also change it to use a .env instead